installDeps.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+# Build script
+# set -o errexit
```

Comment:
Is this file used? Remove commented code.

Reply:
Not yet done.

Reply:
Done